### ISSUE TITLE
Accessibility: PropertyGrid child tables have identical names

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1864]
**Repro steps:**
1. Create .NET Core WinForms app with two PropertyGrid controls and launch it.
3. Start Accessibility insights / Inspect app.
4. Check the names of both tables present on screen.
 
**Actual:**
Issue 1: Both tables have same name "Properties Window".
 
**Expected:**
Issue 1: Both tables should have different names so as to differentiate between them.
 
**User Impact:**
As both tables have same name, screen reader will also announce same which would confuse screen reader users.

DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/956750

**Comment by M-Lipin** [comment]
Fixed by https://github.com/dotnet/winforms/pull/1866.

**Comment by Jessie-Zhang01** [comment]
Verified in .Net core 3.1 latest build 3.1.100.14076 from release/3.1 branch, this bug is fixed. Please see following screenshot: 
![image](https://user-images.githubusercontent.com/45864649/65830488-8b5b1800-e2e2-11e9-80f6-70f811e8bf93.png)

**Comment by M-Lipin** [comment]
DevDiv issue: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/776323
